src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -526,46 +541,67 @@ values are represented by the red spectrum.
 
 ## Accessibility
 
-Accessible design not only helps users with disabilities, it provides better
-user experiences for everyone. The most common form of color blindness is
-red/green color blindness. The inability for some users to distinguish between
-red and green means that products cannot simply rely on color to show status. As
-a result, the status system relies on three key elements; color, shape, and
-symbol.
+### Elements
```

Comment:
> As a result, status indicators should rely on at least two elements: color, shape, or symbol. 

I think this means to say: "As a result, status indicators should rely on at least two of the following elements: color, shape, or symbol."